### DISCUSSION TITLE
inserting/editing node .  black node core background - cannot see text being entered

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#1215] [Help] [answered]
In the attached map, any new node has a black background so I cannot see the text I am entering.  It happens on creating or editing the node - except when editing the root node.

Any idea what is going on?  

![insert node _ black node core](https://github.com/freeplane/freeplane/assets/6380301/72c006a0-10aa-4646-89bc-bbc435884396)
[Bible notes - Copy.zip](https://github.com/freeplane/freeplane/files/11678866/Bible.notes.-.Copy.zip)

**Comment by kenfhill84083** [comment]
> In the attached map, any new node has a black background so I cannot see the text I am entering. It happens on creating or editing the node - except when editing the root node.

forgot to mention that new maps do not have this problem.

**Comment by sifran-github** [comment, marked as answer]
your default node background is transparent.

click ctrl+f11, select default, and click here:
![image](https://github.com/freeplane/freeplane/assets/5730355/bbf503a4-64b3-41e1-b65d-b3e09f070a34)
and click ok.

**Reply by kenfhill84083** [reply]
> your default node background is transparent.
> 
> click ctrl+f11, select default, and click here: ![image](https://user-images.githubusercontent.com/5730355/244200318-bbf503a4-64b3-41e1-b65d-b3e09f070a34.png) and click ok.

Thanks.  I should have waited for this.  I created a new map and copied everything from the old map (much bigger than the sample I posted) except the root node.  I lost some styles but it was past time I did some housecleaning there anyway.

**Comment by sifran-github** [comment]
it might be a bug actually, because... why not allow transparent background on default nodes...? here the background is not black so the node should not be black. My expectation is that it should work even with these settings. Unless I'm missing something.
